lib/membrane_mp4/muxer/cmaf.ex

```diff
@@ -30,138 +36,210 @@ defmodule Membrane.MP4.Muxer.CMAF do
       |> Map.from_struct()
       |> Map.merge(%{
         seq_num: 0,
-        elapsed_time: 0,
-        samples: [],
-        old_input_caps: nil,
-        new_output_caps: nil
+        awaiting_caps: nil,
+        pad_to_track: %{},
```

Comment:
What do you think about renaming `pad_to_track` to `pad_to_track_data`? I saw you refer to values of `pad_to_track` as `track_data` and indeed, `track` ends up being only one of many fields.